db/seeds.rb

```diff
@@ -238,4 +238,146 @@ def create_ranges(metric_configuration_id)
+
+py_mi = NativeMetricSnapshot.create(
+    name: "Maintainability Index", code: "mi", metric_collector_name: "Radon", scope: "PACKAGE",
+    description: "Maintainability Index is a software metric which measures how maintainable (easy to support and change) the source code is. The maintainability index is calculated as a factored formula consisting of SLOC (Source Lines Of Code), Cyclomatic Complexity and Halstead volume."
+)
+
+py_mi_metric_configuration = MetricConfiguration.create(
+  metric_snapshot_id: py_mi.id, weight: 2.0, aggregation_form: "MEAN", reading_group_id: scholar.id,
+  kalibro_configuration_id: python_configuration.id)
+
+# These ranges were chosen by converting the rangesfrom the original Maintainability Index
+# paper [1] from it's scale of 0 to 171 to the one that Radon uses (0 to 100).
+#
+# The original ranges were:
+# * 0-65: difficult to maintain
+# * 65-85: moderately maintainable
+# * > 85: highly maintainable
+#
+# Converted to 0-100 with simple multiplication yields 0, ~38 and ~50.
+#
+# [1] COLEMAN, Don et al. Using metrics to evaluate software system maintainability. Computer, v. 27, n. 8, p. 44-49, 1994.
+[
+  { beginning: 50, end: Float::INFINITY, comments: "", reading_id: readings[:very_good].id },
+  { beginning: 38, end: 50, comments: "", reading_id: readings[:regular].id },
+  { beginning: 0, end: 38, comments: "", reading_id: readings[:bad].id }
+].create_ranges(py_mi_metric_configuration.id)
+
+################################################################################
+
+# Those ranges are used for all Python metrics that count LOC in some way
+loc_ranges = [
+  { beginning: 0, end: 1000, comments: "", reading_id: readings[:good].id },
+  { beginning: 1000, end: 5000, comments: "", reading_id: readings[:regular].id },
+  { beginning: 5000, end: Float::INFINITY, comments: "", reading_id: readings[:bad].id },
+]
+
+py_loc = NativeMetricSnapshot.create(
+    name: "Lines of Code", code: "loc", metric_collector_name: "Radon", scope: "PACKAGE",
+    description: "The total number of lines of code. It is the sum of the SLOC and the number of blank lines: the equation LOC = SLOC + Blanks should always hold."
+)
+py_loc_metric_configuration = MetricConfiguration.create(
+  metric_snapshot_id: py_loc.id, weight: 1.0, aggregation_form: "MEAN", reading_group_id: scholar.id,
+  kalibro_configuration_id: python_configuration.id)
+loc_ranges.create_ranges(py_loc_metric_configuration.id)
+
+################################################################################
+
+py_lloc = NativeMetricSnapshot.create(
+    name: "Logical Lines of Code", code: "lloc", metric_collector_name: "Radon", scope: "PACKAGE",
+    description: "The number of logical lines of code. Every logical line of code contains exactly one statement."
+)
+
+py_lloc_metric_configuration = MetricConfiguration.create(
+  metric_snapshot_id: py_lloc.id, weight: 1.0, aggregation_form: "SUM", reading_group_id: scholar.id,
```

Comment:
This should probably have the same aggregation form as the "loc" MetricConfiguration